src/main/java/dev/jbang/util/JavaUtil.java

```diff
@@ -123,17 +123,19 @@ public static String parseJavaOutput(String output) {
 		return null;
 	}
 
+	// Match the first number in a Java version string, except if
+	// it starts with "1.", in that case, match the second number
+	private static final String JVERSION = "^(?:1\\.)?(\\d+).*";
+
 	public static int parseJavaVersion(String version) {
-		if (version != null) {
-			try {
-				String[] nums = version.split("[-.+]");
-				String num = nums.length > 1 && nums[0].equals("1") ? nums[1] : nums[0];
-				return Integer.parseInt(num);
-			} catch (NumberFormatException ex) {
-				// Ignore
-			}
+		if (version == null)
```

Comment:
We don't use this syntax in the JBang code, we always use { } , even for single lines